<launch>
    <arg name="start_x" default="30" />
    <arg name="start_y" default="100" />


    <include file="$(find acting)/launch/acting.launch">
    </include> <!--This
    also launches control.-->

    <include file="$(find mapping)/launch/mapping.launch">
        <arg name="lidar_discard_probability" value="0.0" />
    </include>

    <node pkg="sim" type="intermediate_layer_integration.py" name="lidar_mock" output="screen" />

    <node pkg="acting" type="trajectory_debug_source.py"
        name="traj_debug_source"
        output="screen">
        <param name="output_topic_name" value="/paf/planning/trajectory" />
        <param
            name="default_source_name" value="lanechange" />
        <param name="trajectory_source"
            value="lanechange" />
        <param name="origin_x" value="$(arg start_x)" />
        <param name="origin_y"
            value="$(arg start_y)" />
    </node>

    <node pkg="sim" type="car_sim.py" name="car_sim" output="screen">
        <param name="car_start_x" value="$(arg start_x)" />
        <param name="car_start_y"
            value="$(arg start_y)" />
    </node>

    <!--
    <node pkg="potential_field" type="p_field_planner.py" name="pot_planner" output="screen">
        <param name="input_topic_name" value="paf/planning/trajectory" />
        <param name="output_topic_name" value="paf/acting/trajectory" />
    </node>
    -->

    <include file="$(find teb_planner)/launch/teb_planner.launch">
    </include>


</launch>